pep_sphinx_extensions/pep_processor/html/pep_html_redirects.py

```diff
@@ -0,0 +1,19 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from pathlib import Path
+
+if TYPE_CHECKING:
+    from collections.abc import Iterator
+
+    from sphinx.application import Sphinx
+
+
+def prepare_redirect_pages(app: Sphinx) -> Iterator[tuple[str, dict, str]]:
+    for path in Path(app.srcdir).glob("pep-????.???"):
```

Comment:
```suggestion
    for path in Path(app.srcdir).glob("pep-[0-9][0-9][0-9][0-9].???"):
```

Use the correct glob to ensure this is actually a PEP with a valid number. Otherwise, it can match any arbitrary `txt` or `rst` file with a four-character suffix, which will cause an error on line 18.

Reply:
> if we ever do

This is I suppose my reluctance to put `[0-9]` rather than `?` -- I seriously think this is very unlikely to happen, and `[0-9]` is five times as many characters as `?`. (The thought applies in all cases that this has been suggested).

A

Reply:
I hope you didn't stay up all night...or just up really early?

A few more characters perhaps, but in terms of what actually matters, does that make it that much less clear to the reader? And is it really worth sacrificing precision and correctness, and risking a potential future hard build failure just to skimp on a few characters? I realize there is _some_ tradeoff to be made, but it seems the rest of us who have weighed in so far have considered it a worthwhile one.

Reply:
Done. I still disagree though!

A